tests/unit/ApprovalType.spec.ts

```diff
@@ -61,14 +65,17 @@ describe('Initialize ApprovalType component', () => {
 
   it('component emits events when court order number entered', async () => {
     const wrapper: Wrapper<ApprovalType> = createDefaultComponent()
+    // Select court order radio to show the court order number text-field
+    const radio = wrapper.find('#court-order-radio')
+    await radio.setChecked(true)
+
     // Input text into text-field
     const input = wrapper.find('#court-order-number-input')
-    input.setValue('89123456')
-    await Vue.nextTick()
+    await input.setValue('89123456')
 
-    expect(wrapper.emitted('emitCourtNumberChange').pop()[0]).toEqual('89123456')
-    expect(wrapper.emitted('emitRadioButtonChange').pop()[0]).toEqual('VIA COURT ORDER')
-    expect(wrapper.emitted('emitValid').pop()[0]).toEqual(true)
+    expect(wrapper.emitted('courtNumberChange')[0][0]).toEqual('89123456')
+    expect(wrapper.emitted('radioButtonChange').pop()[0]).toEqual('VIA COURT ORDER')
```

Comment:
Why do you look at only the last event (using pop())? As this is a fixed test case, you should know how many times this event was emitted before you checked the radio button and set the value, and now you can verify any new expected events.

Ditto line 78.